docs/fidesops/docs/saas_connectors/example_configs/zendesk.md

```diff
@@ -0,0 +1,147 @@
+
+# Zendesk
+
+## Implementation Summary
+Fidesops uses the following Zendesk endpoints to retrieve and delete Personally Identifiable Information (PII) when processing a Data Subject Request (DSR). Right to Access and Right to Delete (Right to Forget) support for each endpoint is noted below. 
+
+|Endpoint | Right to Access | Right to Delete |
+|----|----|----|
+|[Search](https://developer.zendesk.com/api-reference/ticketing/ticket-management/search/) | Yes | No |
+|[Identities](https://developer.zendesk.com/api-reference/ticketing/users/user_identities/) | Yes | No |
+|[Tickets](https://developer.zendesk.com/api-reference/ticketing/tickets/tickets/) | Yes | Yes |
+|[Ticket Comments](https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_comments/) | Yes | Yes |
+
+## Connection Settings
+Fidesops provides a [Postman collection](../../postman/using_postman.md) for easily establishing connections to your third party applications. Additional connection instructions may be found in the [configuration guide](../saas_config.md).
+
+**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `MASKING_STRICT` variable in your `fidesops.toml` file is set to `FALSE`. 
```

Comment:
We recently changed the values in `fidesops.toml` to lower case so it should be `masking_strict`, let's keep an eye out for similar changes, I'm not sure if we updated all of the docs with that change https://github.com/ethyca/fidesops/pull/876